_PlatformOS/getting-started/pages/adding-visuals-pages-liquid.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Adding Visuals to Pages: Liquid
+permalink: /get-started/pages/adding-visuals-pages-liquid
+---
+This guide will help you add visuals (images) to your pages using a Liquid page with the format CSS.  
+
+## Requirements
+To follow the steps in this tutorial, you should be familiar with the required directory structure for your code base, understand the concepts of pages, layouts, and assets, and know how to create a page.  
+
+* [Required Directory Structure]()
+* [Pages]()
+* [Layouts]()
+* [Assets]()
+* [Creating a Page]()
+
+## Steps 
+
+Adding visuals to your pages using Liquid is a three-step process:
+
+1. Prepare an image
+2. Create page `logo.css.liquid`
+3. Link to the `logo.css.liquid` page in your layout
+
+### Step 1: Prepare an image
+Put an image into the `marketplace_builder/assets/images` directory, e.g. logo.svg. 
+
+### Step 2: Create page `logo.css.liquid`
+
+Create a page with the format CSS in your `marketplace_builder/pages` directory named `logo.css.liquid`: 
+
+{% raw %}
+
+```liquid
+---
+slug: logo
+format: css
+---
+
+.logo {
+   background-image: url({{ 'images/logo.svg' | asset_url }});
+
+}
+```
+
+{% endraw %}
+
+Because assets are placed on a CDN, you use the `{{ asset_url }}` helper to get the path of the image. You can't use a relative path, because the path would be relative to the URL of your Instance, not the asset URL on the CDN. 
```

Comment:
There are two things: 
asset_url - variable, which is a hash with mapping asset name to asset url (usage: {{ asset_url }})
asset_url - filter, which is described in docs - https://documentation.near-me.com/reference/liquid-filters-static/ - which we recommend for normal usage